cardano-cli/src/Cardano/CLI/Types.hs

```diff
@@ -75,8 +77,8 @@ data CBORObject = CBORBlockByron Byron.EpochSlots
 
 -- Encompasses stake certificates, stake pool certificates,
 -- genesis delegate certificates and MIR certificates.
-newtype CertificateFile = CertificateFile { unCertificateFile :: FilePath }
-                          deriving newtype (Eq, Show)
+newtype CertificateFile direction = CertificateFile { unCertificateFile :: File direction }
```

Comment:
Can we use `KindSignatures` to make is explicit as to what the type variable should be instantiated to?
e.g `newtype CertificateFile (direction :: FileDirection) = CertificateFile { unCertificateFile :: File direction }`

Reply:
Done!